### PR TITLE
Feat/completed role service

### PR DESCRIPTION
**Pull request by sugiartofahmi**
- [x] Delete role
image

- [x] Create role
image

- [x] Update role
image

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           node-version: 20
           cache: 'pnpm'
-      - run: pnpm install --frozen-lockfile
+      - run: pnpm install --no-frozen-lockfile
       - run: pnpm nx run-many --target=lint ---all --skip-nx-cache
```
Comment (maulanasdqn) [inline .github/workflows/main.yml:20]: ieu --- na loba teuing

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -30,5 +30,5 @@ jobs:
         with:
           node-version: 20
           cache: 'pnpm'
-      - run: pnpm install --frozen-lockfile
+      - run: pnpm install --no-frozen-lockfile
       - run: pnpm nx run-many --target=build ---all --skip-nx-cache
```
Comment (maulanasdqn) [inline .github/workflows/main.yml:34]: --- na loba teuing

**Review by maulanasdqn — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (sugiartofahmi) [reply .github/workflows/main.yml:20]: image

nge fix pas install bang

File: .github/workflows/main.yml (hunk 2 above)
Comment (sugiartofahmi) [reply .github/workflows/main.yml:34]: https://github.com/maulanasdqn/fnb-management/pull/63#discussion_r1646478472

**Review by maulanasdqn — APPROVED** [review]

**Pull request merged**